src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -56,7 +55,7 @@ export class MeetingManager implements AudioVideoObserver {
 
   audioVideoObservers: AudioVideoObserver = {};
 
-  configuration: MeetingSessionConfiguration | null = null;
+  meetingSessionConfiguration: MeetingSessionConfiguration | null = null;
```

Comment:
Prefer using `undefined` instead of `null`.

Reply:
Could you also update below method to make it consistent?
```ts
createLogger(configuration: MeetingSessionConfiguration): Logger {
```

Reply:
Done